packages/server/src/jsRunner/vm/isolated-vm.ts

```diff
@@ -268,7 +227,10 @@ export class IsolatedVM implements VM {
   private getFromContext(key: string) {
     const ref = this.vm.global.getSync(key, { reference: true })
     const result = ref.copySync()
-    ref.release()
+
+    new Promise(() => {
+      ref.release()
+    })
```

Comment:
Not sure I understand the purpose of this. What does it fix?

Reply:
Having it async added some unnecessary delay on the process. This does it async, not really caring if it fails. Anyways, the garbage collector will pick it up, this just speeds up the process disposing it